projects/catb.org/wumpus/package.yml

```diff
@@ -0,0 +1,27 @@
+distributable:
+  url: http://www.catb.org/~esr/wumpus/wumpus-1.7.tar.gz
+  strip-components: 1
+
+# When https://gitlab.com/esr/wumpus/-/merge_requests/3 is merged, then switch to
+#   https://gitlab.com/esr/wumpus/-/archive/{{ version.raw }}/wumpus-{{ version.raw }}.tar.gz
+
+versions:
+  - 1.7
+
+# TODO: Is there a gitlab version provider?
+
+provides:
+  - bin/wumpus
+  - bin/superhack
+
+build:
+  dependencies:
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+
+  script: |
+    make
+    make prefix={{prefix}} install
+
+test:
+  script: echo no way to test this
```

Comment:
lol `true` with a comment is what we do, but this is fine.

Though there must be some way, even if it's just execute in bg then kill after a few seconds. But no need to go the extra mile today.

Reply:
How would you tell if it worked if you execute it in the background? Check to see if the pid exists?